src/rt-app.c

```diff
@@ -639,10 +639,9 @@ int run(thread_data_t *tdata,
 			return perf;
 
 		log_debug("[%d] runs events %d type %d ", ind, i, events[i].type);
-		if (opts.ftrace)
-				log_ftrace(ft_data.marker_fd,
-						"[%d] executing %d",
-						ind, i);
+		log_ftrace(ft_data.marker_fd, FTRACE_EVENT,
+			   "rtapp_event: id=%d type=%d desc=%s",
```

Comment:
It would be nice to also print duration for some of the event types (e.g., run, sleep, timer).
Wonder if that clutters code too much, though?

Reply:
I guess we can add one category to track this more detailed info.
However, what we have so far with the current set of events, it's good enough to compute the duration of each event in post-processing. We have indeed the trace timestamp and a start/stop marker for each command.